docs/monthly/2023-05-monthly.md

```diff
@@ -0,0 +1,250 @@
+
+#### Release 0.10.0
+
+- First mainnet compatible release of `hydra-node` including technical changes and documentation
+
+- Mainnet exposure currently limited to 100₳ per participant
+
+- Added some missing features to the client API (short-term solutions)
+
+- Some quality of life improvements in running the `hydra-node`
+
+- [Full release notes](https://github.com/input-output-hk/hydra/releases/tag/0.10.0) and list of [delivered features](https://github.com/input-output-hk/hydra/milestone/8?closed=1)
+
+#### Notable roadmap updates
+
+- Got new input and concrete scenario for benchmarking (off-chain) performance
+  of a Hydra Head. As it is a very basic one, we conretized the item from an
+  idea to a feature and started work on it.
+
+- Added a new feature item to add Hydra as tool to developer platforms within
+  our agenda of [promoting Hydra as an open-source
+  platform](https://hydra.family/head-protocol/monthly/2023-01#themes-for-2023)
+  for scalability on Cardano.
+
+- Already completed a first feature for 0.11.0 and considering to release it as
+  early as possible with latest advances.
+
+![](./img/2023-05-roadmap.png) <small><center>The latest roadmap with features and ideas</center></small>
+
+## Development
+
+[Issues and pull requests closed since last
+report](https://github.com/input-output-hk/hydra/issues?q=is%3Aclosed+sort%3Aupdated-desc+closed%3A2023-04-26..2023-05-24)
+
+This month, the team worked on the following:
+
+#### Timed transactions [#196](https://github.com/input-output-hk/hydra/issues/196#)
+
+To realize the Hydra Head protocol, a `hydra-node` observes and follows the
+chain progress on the layer 1 (L1). Until now, this component was only observing
+protocol related transactions. Now, on every block roll forward, a `Tick` event
+carrying the notion of time from L1 is observed by the `hydra-node` and is used
+to validate transactions in a Hydra head. This means that clients can submit
+time bounded transactions to an open head and expect them to be validated using
+the same slot that would be used on L1. It is important to mention, that time
+only advances on L1 when a block was produced.
+
+![](./img/2023-05-timed-transactions.jpg) <small><center>Timed transactions in a Hydra Head state channel</center></small>
+
+This feature will make the Hydra L2 ledger now en-par with UTxO features
+available on the Cardano L1. A logical next step in this direction could be to
+make time in the state channel advance every configured `slotLength` (e.g. every
+second) using the system clock in between blocks as a form of [dead reckoning](https://en.wikipedia.org/wiki/Dead_reckoning).
+
+#### Removing --ledger-genesis [#863](https://github.com/input-output-hk/hydra/pull/863)
+
+The hydra-node was taking two command line options to configure the ledger used
+to validate transactions on layer 2: `--ledger-protocol-parameters` and
```

Comment:
consider saying layer 2 (L2) - for consistency with the first layer 1 (L1) mention

Reply:
```suggestion
to validate transactions on layer 2 (L2): `--ledger-protocol-parameters` and
```